packages/ui5-middleware-onelogin/src/cookieGetter.ts

```diff
@@ -207,16 +276,12 @@ export default class CookieGetter {
 					isLoginPage = await this.isLoginPage(page);
 					if (!isLoginPage) {
 						break;
-					} else {
-						if (effectiveOptions.configuration.debug) {
-							log.info(`"${attr.url}" looks like a login page, reloading...`);
-						}
+					} else if (effectiveOptions.configuration.debug) {
+						log.info(`"${attr.url}" looks like a login page, reloading...`);
 					}
 				}
-				if (isLoginPage) {
-					if (effectiveOptions.configuration.debug) {
-						log.info(`Couldn't login using a certificate!`);
-					}
+				if (isLoginPage && effectiveOptions.configuration.debug) {
+					log.info(`Couldn't login using a certificate!`);
```

Comment:
Not sure about the cleanup here. It does make sense in its current form. I do believe however, that it was "designed" like this to have explicit entry points for further handling later on and just in case someone has debug enable, to log it as well.

If nobody is against it though (as there ... is no handling currently anway :D) I'm all for it. :)

Reply:
true, lets leave it like that, i reverted that change

Reply:
https://github.com/ui5-community/ui5-ecosystem-showcase/pull/1079/commits/3d33fb5dbb00a5c2eb2f470e30cf0437d6381dbe